frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -113,12 +113,14 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
           <h5>Parcel Details</h5>
         </Col>
         <Col md={6}>
-          <PidPinForm
-            nameSpace={nameSpace}
-            handlePidChange={noop}
-            handlePinChange={noop}
-            disabled={disabled}
-          />
+          {propertyTypeId !== PropertyTypes.SUBDIVISION && (
```

Comment:
Since the api handles supplying the subdivision with the correct PID/PIN, simply hide those form controls.